www/src/main/java/org/wso2/integration/ballerina/SiteBuilder.java

```diff
@@ -62,9 +72,14 @@
 public class SiteBuilder {
     // Setup logger.
     private static final Logger logger = Logger.getLogger(SiteBuilder.class.getName());
+    // Current commit hash.
+    private static String commitHash = null;
```

Comment:
Shall we get rid of this variable and pass the value to processReadmeFile method?

Reply:
I added this variable because we only need to get the commit hash only once. This won't be changed, So no need to call getCommitHashByReadingGitProperties(...) method inside processReadmeFile(...) method every time, it processing a file.